config.dev.toml

```diff
@@ -43,6 +43,14 @@ LogLevel = "debug"
 # Env override: ATHENS_CLOUD_RUNTIME
 CloudRuntime = "none"
 
+# ActivateDebugHandler specifies if the pprof endpoints should be exposed.
+# Note that this option is not meant to be activated forever on a server
+# and should be desactivated once not needed.
+ActivateDebugHandler = false
+
+# DebugHandlerPort specifies the port on which pprof will be exposed if activated.
+DebugHandlerPort = ":8081"
```

Comment:
same as above, we can just call it `PprofPort`.

Also since we run at port 3000 by default, we should make this port 3001. Unles 8081 is a convention port for pprof? Not sure :)

Reply:
No problem, no it's not a convention.